internal/abi/fastly/hostcalls_noguest.go

```diff
@@ -1,5 +1,4 @@
-//go:build (!tinygo.wasm && !wasi) || nofastlyhostcalls
-// +build !tinygo.wasm,!wasi nofastlyhostcalls
+//go:build !((tinygo.wasm && wasi) || wasip1) || nofastlyhostcalls
```

Comment:
Could you also make this change to the tests under `integration_tests` as well?

Reply:
Done